commonTest/src/main/scala/pl/touk/nussknacker/engine/management/common/StreamingDockerTest.scala

```diff
@@ -39,16 +40,35 @@ trait StreamingDockerTest extends DockerTest with Matchers {
 
   protected lazy val deploymentManager: DeploymentManager = {
     val typeConfig = ProcessingTypeConfig.read(config)
-    deploymentManagerProvider.createDeploymentManager(ModelData(typeConfig), typeConfig.deploymentConfig)
+    val modelDependencies: ModelDependencies = {
+      ModelDependencies(
+        Map(),
+        componentId => DesignerWideComponentId(componentId.toString),
+        workingDirectoryOpt = None,
+        _ => true
+      )
+    }
+    val deploymentManagerDependencies = DeploymentManagerDependencies(
+      new ProcessingTypeDeploymentServiceStub(List.empty),
+      actorSystem.dispatcher,
+      actorSystem,
+      backend
+    )
+    deploymentManagerProvider.createDeploymentManager(
+        ModelData(typeConfig, modelDependencies),
+        deploymentManagerDependencies,
+        typeConfig.deploymentConfig,
+        None
+      ).valueOr(err => throw new IllegalStateException(s"Invalid Deployment Manager: ${err.toList.mkString(", ")}"))
   }
 
   protected def deployProcessAndWaitIfRunning(process: CanonicalProcess, processVersion: ProcessVersion, savepointPath: Option[String] = None): Assertion = {
     implicit val freshnessPolicy: DataFreshnessPolicy = DataFreshnessPolicy.Fresh
     deployProcess(process, processVersion, savepointPath)
 
     eventually {
-      val jobStatus = deploymentManager.getProcessStates(ProcessName(process.id)).futureValue
-      logger.info(s"Waiting for deploy: ${process.id}, $jobStatus")
+      val jobStatus = deploymentManager.getProcessStates(process.name).futureValue
+      logger.info(s"Waiting for deploy: ${process.name.value}, $jobStatus")
```

Comment:
Omit `.value` - toString produces the same

Reply:
Ok, I was used to the default `toString` implementation for case classes.